github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-postsubmits.yaml

```diff
@@ -46,6 +97,19 @@ postsubmits:
           - "-c"
           - >
             cat $QUAY_PASSWORD | podman login --username $(<$QUAY_USER) --password-stdin quay.io &&
+            SHORT_SHA=$(git rev-parse --short HEAD) &&
+            GCS_FILE_PATH=gs://kubevirt-prow/release/kubevirt/kubevirtci/s390x-$SHORT_SHA &&
```

Comment:
I would prefer if the s390x job could relay on the tag from the x86 image rather than relying on the s390x job here.

Reply:
added reply to same comment in other place. Once you confirm, I will do change here also.